### ISSUE TITLE
prey_1.5.0_amd64.deb installer is not working for Debian 8

### ISSUE THREAD

**Issue by cdipietro** [prey/prey-node-client#194]
Hi,

I have a 64 bits Debian 8 with Gnome 3. I have tried to install Prey 1.5.0 using the [.deb installer](https://github.com/prey/prey-node-client/releases/download/v1.5.0/prey_1.5.0_amd64.deb) as follows:

```bash
carlos@lara:~$ sudo gdebi prey_1.5.0_amd64.deb 
Reading package lists... Done
Building dependency tree        
Reading state information... Done
Building data structures... Done 
Building data structures... Done 
Requires the installation of the following packages: giblib1 libid3tag0 libimlib2 mpg123 scrot streamer xawtv-plugins 

Utility for tracking stolen or missing computers
 Prey is a lightweight program that will help you find, lock and recover your
 computer should it ever get lost or stolen.
Do you want to install the software package? [y/N]:y
Fetched 0 B in 0s (0 B/s)                                                                                                                                                                            
Selecting previously unselected package libid3tag0.
(Reading database ... 246732 files and directories currently installed.)
Preparing to unpack .../libid3tag0_0.15.1b-11_amd64.deb ...
Unpacking libid3tag0 (0.15.1b-11) ...
Selecting previously unselected package libimlib2.
Preparing to unpack .../libimlib2_1.4.6-2+b3_amd64.deb ...
Unpacking libimlib2 (1.4.6-2+b3) ...
Selecting previously unselected package giblib1:amd64.
Preparing to unpack .../giblib1_1.2.4-10_amd64.deb ...
Unpacking giblib1:amd64 (1.2.4-10) ...
Selecting previously unselected package xawtv-plugins.
Preparing to unpack .../xawtv-plugins_3.103-3+b1_amd64.deb ...
Unpacking xawtv-plugins (3.103-3+b1) ...
Selecting previously unselected package mpg123.
Preparing to unpack .../mpg123_1.20.1-2_amd64.deb ...
Unpacking mpg123 (1.20.1-2) ...
Selecting previously unselected package scrot.
Preparing to unpack .../scrot_0.8-13_amd64.deb ...
Unpacking scrot (0.8-13) ...
Selecting previously unselected package streamer.
Preparing to unpack .../streamer_3.103-3+b1_amd64.deb ...
Unpacking streamer (3.103-3+b1) ...
Processing triggers for mime-support (3.58) ...
Processing triggers for man-db (2.7.0.2-5) ...
Setting up libid3tag0 (0.15.1b-11) ...
Setting up libimlib2 (1.4.6-2+b3) ...
Setting up giblib1:amd64 (1.2.4-10) ...
Setting up xawtv-plugins (3.103-3+b1) ...
Setting up mpg123 (1.20.1-2) ...
update-alternatives: using /usr/bin/mpg123.bin to provide /usr/bin/mpg123 (mpg123) in auto mode
update-alternatives: using /usr/bin/mpg123.bin to provide /usr/bin/mp3-decoder (mp3-decoder) in auto mode
Setting up scrot (0.8-13) ...
Setting up streamer (3.103-3+b1) ...
Processing triggers for libc-bin (2.19-18+deb8u3) ...
Selecting previously unselected package prey.
(Reading database ... 246883 files and directories currently installed.)
Preparing to unpack prey_1.5.0_amd64.deb ...
Unpacking prey (1.5.0) ...
Setting up prey (1.5.0) ...
Creating user prey...
Creating a user called prey
prey already seems to have impersonation privileges. Skipping...
Creating config dir: /etc/prey
Touching log file: /var/log/prey.log
Setting permissions on /usr/lib/prey
chowning /usr/lib/prey/versions/1.5.0/README.md to uid 999 and gid 999
chowning /usr/lib/prey/versions/1.5.0/license.txt to uid 999 and gid 999

[ ... MANY "chowning"s AFTER ... ]

chowning /usr/lib/prey/versions/1.5.0 to uid 999 and gid 999
chowning /usr/lib/prey/versions to uid 999 and gid 999
chowning /usr/lib/prey to uid 999 and gid 999
Setting permissions on /etc/prey
chowning /etc/prey to uid 999 and gid 999
Setting permissions on /var/log/prey.log
chowning /var/log/prey.log to uid 999 and gid 999
Running "config activate" as prey
--
Ensuring presence of config dir: /etc/prey
Syncing /etc/prey/prey.conf with /usr/lib/prey/versions/1.5.0/prey.conf.default
Setting up this as current...
No version is set as current.
Setting permissions on /usr/lib/prey/current
Running post_activate hooks...
Voila! All good.
--
Installing init scripts.
Sweet! Please run `prey config gui` to finish installation.
Voila! All good.
localuser:prey being added to access control list
```

But at this point, it appears a small window saying:

> Failed to run bin/prey config gui as user prey.
>
>Failed to communicate with gksu-run-helper.
>
>Received:
>
>While expecting:
> gksu: waiting

And after closing it, the installation process concludes as follows:

```bash
dpkg: error processing package prey (--install):
 subprocess installed post-installation script returned error exit status 3
Errors were encountered while processing:
 prey
```

So far I am still unable to install the program by using the .deb installer. I was hoping you could give me any hint on what could I do to make the installer work.

Thanks in advance.

**Comment by alberthdev** [comment]
I can also confirm that the latest 1.5.1 release of Prey does not install correctly, either... same output as above (trimmed to just show the end):

    Running "config activate" as prey
    --
    Ensuring presence of config dir: /etc/prey
    Syncing /etc/prey/prey.conf with /usr/lib/prey/versions/1.5.1/prey.conf.default
    Setting up this as current...
    Warning: This version is already set as current.
    Setting permissions on /usr/lib/prey/current
    Running post_activate hooks...
    Voila! All good.
    --
    Installing init scripts.
    Sweet! Please run `prey config gui` to finish installation.
    Voila! All good.
    localuser:prey being added to access control list
    dpkg: error processing package prey (--configure):
     subprocess installed post-installation script returned error exit status 1
    E: Sub-process /usr/bin/dpkg returned an error code (1)

**Comment by mhalano** [comment]
@alberthdev 
What you mention is a separated bug (a regression, actually) and I already submit a issue report about that.

Cheers!

**Comment by javo** [comment]
Hi @cdipietro, I succeeded installing prey 1.5.0 in a 64 bits Debian 8 system.

```bash
prey@dev-deb-s-jessie-1:~$ sudo gdebi prey_1.5.0_amd64.deb

We trust you have received the usual lecture from the local System
Administrator. It usually boils down to these three things:

    #1) Respect the privacy of others.
    #2) Think before you type.
    #3) With great power comes great responsibility.

[sudo] password for prey: 
Reading package lists... Done
Building dependency tree        
Reading state information... Done
Building data structures... Done 
Building data structures... Done 
Requires the installation of the following packages: giblib1 libid3tag0 libimlib2 scrot streamer xawtv-plugins 

Utility for tracking stolen or missing computers
 Prey is a lightweight program that will help you find, lock and recover your
 computer should it ever get lost or stolen.
Do you want to install the software package? [y/N]:y
Get:1 http://security.debian.org/ jessie/updates/main libimlib2 amd64 1.4.6-2+deb8u1 [178 kB]                  
Get:2 http://mirrors.kernel.org/debian/ jessie/main libid3tag0 amd64 0.15.1b-11 [35.3 kB]                      
Get:3 http://mirrors.kernel.org/debian/ jessie/main giblib1 amd64 1.2.4-10 [19.7 kB]                           
Get:4 http://mirrors.kernel.org/debian/ jessie/main xawtv-plugins amd64 3.103-3+b1 [83.7 kB]                   
Get:5 http://mirrors.kernel.org/debian/ jessie/main scrot amd64 0.8-13 [19.3 kB]                               
Get:6 http://mirrors.kernel.org/debian/ jessie/main streamer amd64 3.103-3+b1 [80.9 kB]                        
Fetched 417 kB in 0s (0 B/s)                                                                                   
Selecting previously unselected package libid3tag0.
(Reading database ... 143706 files and directories currently installed.)
Preparing to unpack .../libid3tag0_0.15.1b-11_amd64.deb ...
Unpacking libid3tag0 (0.15.1b-11) ...
Selecting previously unselected package libimlib2.
Preparing to unpack .../libimlib2_1.4.6-2+deb8u1_amd64.deb ...
Unpacking libimlib2 (1.4.6-2+deb8u1) ...
Selecting previously unselected package giblib1:amd64.
Preparing to unpack .../giblib1_1.2.4-10_amd64.deb ...
Unpacking giblib1:amd64 (1.2.4-10) ...
Selecting previously unselected package xawtv-plugins.
Preparing to unpack .../xawtv-plugins_3.103-3+b1_amd64.deb ...
Unpacking xawtv-plugins (3.103-3+b1) ...
Selecting previously unselected package scrot.
Preparing to unpack .../scrot_0.8-13_amd64.deb ...
Unpacking scrot (0.8-13) ...
Selecting previously unselected package streamer.
Preparing to unpack .../streamer_3.103-3+b1_amd64.deb ...
Unpacking streamer (3.103-3+b1) ...
Processing triggers for man-db (2.7.0.2-5) ...
Setting up libid3tag0 (0.15.1b-11) ...
Setting up libimlib2 (1.4.6-2+deb8u1) ...
Setting up giblib1:amd64 (1.2.4-10) ...
Setting up xawtv-plugins (3.103-3+b1) ...
Setting up scrot (0.8-13) ...
Setting up streamer (3.103-3+b1) ...
Processing triggers for libc-bin (2.19-18+deb8u4) ...
Selecting previously unselected package prey.
(Reading database ... 143817 files and directories currently installed.)
Preparing to unpack prey_1.5.0_amd64.deb ...
Unpacking prey (1.5.0) ...
Setting up prey (1.5.0) ...
Creating user prey...
prey user already exists!
Giving prey user passwordless sudo priviledges...
Creating config dir: /etc/prey
Touching log file: /var/log/prey.log
Setting permissions on /usr/lib/prey
```
...
```bash
Setting permissions on /etc/prey
chowning /etc/prey to uid 1000 and gid 1000
Setting permissions on /var/log/prey.log
chowning /var/log/prey.log to uid 1000 and gid 1000
Running "config activate" as prey
--
Ensuring presence of config dir: /etc/prey
Syncing /etc/prey/prey.conf with /usr/lib/prey/versions/1.5.0/prey.conf.default
Setting up this as current...
No version is set as current.
Setting permissions on /usr/lib/prey/current
Running post_activate hooks...
Voila! All good.
--
Installing init scripts.
Sweet! Please run `prey config gui` to finish installation.
Voila! All good.
localuser:prey being added to access control list
Gtk-Message: Failed to load module "canberra-gtk-module"
Firing up config GUI!
All done!

```

Try removing that version and installing the 1.5.1 with all the necessary dependencies and then let me know if it worked. :)

**Comment by alberthdev** [comment]
Seems to be fixed on Debian 8, with the latest 1.6.1 version.
